content/en/providers/check-in/sp/_index.md

```diff
@@ -1919,20 +1920,20 @@ connected to Check-in.
-|   **SAML Attribute(s)** | <ul><li>`urn:oid:1.3.6.1.4.1.5923.1.1.1.9` (eduPersonScopedAffiliation)</li><li>`urn:oid:1.3.6.1.4.1.25178.4.1.11` (voPersonExternalAffiliation)</li></ul> |
-|          **OIDC scope** | `eduperson_scoped_affiliation`                                                                                                                             |
-|       **OIDC claim(s)** | `eduperson_scoped_affiliation`                                                                                                                             |
-| **OIDC claim location** | <ul><li>UserInfo Endpoint</li><li>Introspection Endpoint</li></ul>                                                                                         |
-|              **origin** | Check-in assigns this attribute on user registration                                                                                                       |
-|             **changes** | Yes                                                                                                                                                        |
-|        **multiplicity** | Multi-valued                                                                                                                                               |
-|        **availability** | Always                                                                                                                                                     |
-|             **example** | <ul><li>`member@example.org`</li><li>`faculty@example.org`</li></ul>                                                                                       |
-|               **notes** | Service Providers are encouraged to validate the scope of this attribute                                                                                   |
-|              **status** | Stable                                                                                                                                                     |
+|          attribute name | Affiliation                                                              |
+| ----------------------: | :----------------------------------------------------------------------- |
+|         **description** | The user's affiliation within a particular security domain (scope)       |
+|   **SAML Attribute(s)** | `urn:oid:1.3.6.1.4.1.25178.4.1.11` (voPersonExternalAffiliation)         |
+|          **OIDC scope** | `eduperson_scoped_affiliation`                                           |
+|       **OIDC claim(s)** | `eduperson_scoped_affiliation`                                           |
+| **OIDC claim location** | <ul><li>UserInfo Endpoint</li><li>Introspection Endpoint</li></ul>       |
+|              **origin** | Check-in assigns this attribute on user registration                     |
```

Comment:
When available, this information originates from the user's authenticating Identity Provider

Reply:
done